backend/framework/qlf/dashboard/config_file.py

```diff
@@ -0,0 +1,28 @@
+import os
+import shutil
+
+qlf_root = os.getenv('QLF_ROOT')
+config_path = '{}/framework/config/qlf.cfg'.format(qlf_root)
+
+
+def edit_qlf_config_file(keys, values):
```

Comment:
We could use [configparser](https://docs.python.org/3/library/configparser.html) to edit qlf.cfg

Reply:
Ok, I'll add a TODO here for latter.